packages/superset-ui-plugins-demo/package.json

```diff
@@ -9,7 +9,8 @@
     "demo:build": "build-storybook -o _gh-pages",
     "demo:publish": "gh-pages -d _gh-pages",
     "deploy-demo": "npm run demo:clean && npm run demo:build && npm run demo:publish && npm run demo:clean",
-    "storybook": "start-storybook -p 9001"
+    "storybook": "start-storybook -p 9001",
+    "build": "cd ../../ && yarn build && cd - && yarn storybook"
```

Comment:
Much better! Maybe one final nit, I'd name this something like `build-and-run` or maybe follow the example with `demo`, naming the old `storybook` command `storybook:run` and this one just `storybook`

Reply:
Addressed commit lint issues (I think) & renamed the script name to "storybook" - thank you for the suggestions!